paasta_tools/kubernetes_tools.py

```diff
@@ -3502,14 +3502,19 @@ def get_kubernetes_secret(
     secret_name: str,
     service_name: str,
     namespace: str = "paasta",
+    decode: bool = True,
 ) -> str:
```

Comment:
is this now `-> Union[str, bytes]` since we're not always decoding?

Reply:
i'm honestly surprised that mypy didn't catch this - https://github.com/python/typeshed/blob/main/stdlib/base64.pyi#L30 says that `base64.b64decode` returns bytes, so i'm not sure what went wrong with the types here

Reply:
oh, whoops! there are failing mypy tests that run in github actions, they just don't run on `make test` or any other Makefile target afaict

Reply:
oh, 🤦🏻  - i've been meaning to fix that but doing so always gets back-burnered

Reply:
Fixed the mypy type hints